src/web/services/status-message.service.ts

```diff
@@ -1,5 +1,5 @@
-import { Injectable } from '@angular/core';
-import { Observable, Subject } from 'rxjs';
+import { Injectable, TemplateRef } from '@angular/core';
+import { MatSnackBar } from '@angular/material';
 import { StatusMessage } from '../app/components/status-message/status-message';
```

Comment:
Do we still need this? Can remove `app/components/status-message/status-message` too.

Reply:
I'm just using it as the type of the parameter for `showMessage`. I suppose we can do away with it, but I think it looks neater this way.